server/src/main/java/org/tctalent/server/api/admin/DuolingoCouponAdminApi.java

```diff
@@ -61,29 +62,35 @@ public Map<String, Object> importCouponsFromCsv(@RequestParam("file") MultipartF
 
       return Map.of("status", "success", "message", "Coupons imported successfully.");
     } catch (RuntimeException e) {
-      log.error("Failed to import coupons from CSV", e);
-
+      LogBuilder.builder(log)
+          .action("importCouponsFromCsv")
+          .message("Failed to import coupons from CSV")
+          .logInfo();
```

Comment:
Thanks for switching this to the LogBuilder @EhsanEhrari. However, instead of .logInfo(), can you please use .logError(e) for error logging?